### ISSUE TITLE
Signed packages

### ISSUE THREAD

**Issue by axeoth** [tailhook/vagga#149]
Hi,

 Currently, installing Ubuntu packages requires requires overwriting the security settings, by explicitly allowing to install unsigned/unauthenticated packages:

http://vagga.readthedocs.org/en/latest/installation.html#ubuntu

Would it be possible to have signed packages?

 Thank you.

**Comment by tailhook** [comment]
By the way, you can use the following in the apt-sources.list, to skip confirmation on each upgrade:
```
deb [arch=amd64 trusted=yes] http://ubuntu.zerogw.com vagga main
```

**Comment by tekacs** [comment]
@tailhook Just one more request to either provide signed packages or (even through a gateway), provide HTTPS on ubuntu.zerogw.com (I assume it's running your zerogw server, so I can see how that could be difficult without a reverse proxy like Cloudflare :confused:).

With both HTTPS and package signing off, the current installation process is painfully insecure, to the point that I daren't even suggest it as an option in something low-priority (was going to try integrating it to the integer32 playpen), by this approach.

**Comment by tailhook** [comment]
Yes, you're right. I'll try to move it to HTTPS in a week or so (should be a little bit easier than package signing).

**Comment by tailhook** [comment]
Okay, `https://ubuntu.zerogw.com` is up and running. I'm not going to enable HSTS or redirects. I mean it will be accessible both by HTTP and HTTPS at least for some time.

Will upgrade `files.zerogw.com` (scripts and static binaries) shortly too.

**Comment by tekacs** [comment]
Thanks! I guess I can understand not enabling HSTS/redirects for now, at least so people have time to migrate any existing uses of vagga (and since the docs are updated, for new users).

Again, thanks for taking the time!

**Comment by tailhook** [comment]
BTW, I'm curious about your integration with integer32 playpen. I would be cool if you share some details in chat or just contact me privately.
